includes/class-events.php

```diff
@@ -450,81 +360,30 @@ private function reset_event_lock( $event ) {
 	/**
 	 * Turn the event action into a string that can be used with a lock
 	 *
-	 * @param object $event Event data.
+	 * @param Event|stdClass $event
 	 * @return string
 	 */
-	public function get_lock_key_for_event_action( $event ) {
+	public function get_lock_key_for_event_action( $event ): string {
```

Comment:
This method is public, and currently used in the lock class, so have to keep BC for now.

In the future I think either locks because first-class citizens of the Event object, (i.e. `$event->get_lock_key()`), or they are going to get a rework / maybe removed altogether. So just leaving more or less as is for now.